Jenkinsfile

```diff
@@ -29,7 +29,7 @@ pipeline {
    booleanParam(name: 'RUN_BUILD', description: 'if false skip building', defaultValue: true)
    booleanParam(name: 'RUN_BUILD_UPSTREAM', description: 'if false skip building', defaultValue: true)
    string(name: 'SANITYCHECK_RETRY_NUM', description: 'Default number of sanitycheck retries', defaultValue: '7')
-   string(name: 'PLATFORMS', description: 'Default Platforms to test', defaultValue: 'nrf9160_pca10090 nrf9160_pca10090ns nrf52_pca10040 nrf52840_pca10056')
+   string(name: 'PLATFORMS', description: 'Default Platforms to test', defaultValue: 'nrf9160dk_nrf9160 nrf9160_nrf9160ns nrf52dk_nrf52832 nrf52840dk_nrf52840')
```

Comment:
The second one should be nrf9160**dk**_nrf9160ns.

Reply:
corected